plugin.video.jafc/resources/lib/plugin.py

```diff
@@ -0,0 +1,300 @@
+# -*- coding: utf-8 -*-
+"""Main plugin file - Handles the various routes"""
+
+__author__ = "fraser"
+
+import routing
+import xbmc
+import xbmcaddon as xa
```

Comment:
It's good coding practice to leave these imports as they are named; This makes for easy code comprehension.

Reply:
This seems arbitrary rather than objective - the other three plugins I have submitted that have been accepted use *exactly* the same code.

Reply:
I wouldn't classify it as "arbitrary" submission rules do state we follow pep8 guidelines. I wouldn't hold back your submission because of this; I was merely letting you know so in the future you can keep it in mind. =)

Reply:
Hi,

which pep8 guideline says not to use aliases for imports?
Using pycharm and it doesn't flag it and I can't see any reference to this in the guide either...

https://www.python.org/dev/peps/pep-0008/#imports

Am I missing something?

Reply:
Ignore this one